.github/workflows/integration.yml

```diff
@@ -5,7 +5,7 @@ on:
       - 'v*'
     branches:
       - master
-  pull_request_target:
+  pull_request:
```

Comment:
Should be reverted before merging. Same below

Reply:
Sure, they are in a separate commit to remove them easily 👍